guides/micronaut-openapi-generator-server/java/src/main/java/example/micronaut/BookEntity.java

```diff
@@ -0,0 +1,20 @@
+package example.micronaut;
+
+import example.micronaut.model.BookAvailability;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.data.annotation.GeneratedValue;
+import io.micronaut.data.annotation.Id;
+import io.micronaut.data.annotation.MappedEntity;
+import io.micronaut.data.annotation.MappedProperty;
+
+import jakarta.validation.constraints.NotNull;
+
+@MappedEntity("book") // <1>
+public record BookEntity(
+    @Nullable @Id @GeneratedValue(GeneratedValue.Type.AUTO) Long id, // <2>
+    @NonNull @NotNull String name,
```

Comment:
```suggestion
    @NonNull @NotBlank String name,
```

Please, use @NotBlank instead of `@NotNull` for strings.

Reply:
Fixed in 299fb2e5a6